docs/src/fraction.md

```diff
@@ -301,10 +301,6 @@ julia> h = gcd(f, g)
 
 ### Square root
 
-```@docs
-is_square{T <: RingElem}(::FracElem{T})
-```
-
 ```@docs
 Base.sqrt(::FracElem{T}) where {T <: RingElem}
 ```
```

Comment:
By the looks of it, the same treatment should be extended to the `Base.sqrt` methods -- and then perhaps instead of removing the sections here outright, we could something like this:
```suggestion
Methods for `is_square` and `sqrt` are provided for inputs of type `FracElem`.
```